ecosystem/sep-0031.md

```diff
@@ -279,7 +279,7 @@ This is the successful case where a Receiving Anchor confirms that they can fulf
 
 Name | Type | Description
 -----|------|------------
-`id` | string | The persistent identifier to check the status of this payment.
+`id` | string | The persistent identifier to check the status of this payment transaction.
```

Comment:
❔ What distinction are we making by specifying this as the `Stellar payment transaction` vs the `Stellar payment`? We refer to the payment as simply a Stellar payment elsewhere in the doc, like two lines down.

Reply:
For this SEP there's kind of an intersection between payment and transaction definitions even though they are distinct Stellar concepts.

The motivation for that change is the text said this is a "payment id" but it's returned from the `/transactions` endpoint, so it made sense from a syntax perspective to clarify that. That change may be unnecessary though.